### PR TITLE
adding config file for filtering warnings

### PR DESCRIPTION
**Pull request by kzqureshi**
'pytest.ini' file has been added to filter the DeprecationWarning

### CODE REVIEW

**Comment by lang-m** [comment]
@kzqureshi Could you please move the pytest configuration to `pyproject.toml` (https://docs.pytest.org/en/7.1.x/reference/customize.html#pyproject-toml)

File: micromagneticdata/abstract_drive.py [hunk 1]
```diff
@@ -107,7 +107,9 @@ def m0(self):
         Field(...)
 
         """
-        return df.Field.from_file(self._m0_path)
+        field = df.Field.from_file(self._m0_path)
+        field.valid = "norm"
+        return field
```
Comment (lang-m) [inline micromagneticdata/abstract_drive.py:112]: I would like to move this to a separate PR.

File: pytest.ini [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[pytest]
+python_files = test_*.py
+python_functions = test_*
+filterwarnings =
+    ignore::DeprecationWarning:traitlets.*:
```
Comment (lang-m) [inline pytest.ini:5]: [Note for myself] re-check if the warning is suppressed locally, may need further changes (`pytest -W error -W 'ignore: :DeprecationWarning'` seems to work, the warning is raised from dependencies from k3d)

File: pytest.ini [hunk 3]
```diff
@@ -0,0 +1,5 @@
+[pytest]
+python_files = test_*.py
+python_functions = test_*
```
[the remark this answers is not in the document]
Comment (kzqureshi) [reply pytest.ini:3]: yes in `pytest.ini` these are required because these two lines are selecting all the files with test names and then applying the ignore command on the tests.

**Comment by lang-m** [comment]
Closing in favour of #66
